x-pack/plugin/watcher/qa/rest/build.gradle

```diff
@@ -52,13 +52,9 @@ tasks.named("transformV7RestTests").configure({ task ->
 
 tasks.named("yamlRestCompatTest").configure {
   systemProperty 'tests.rest.blacklist', [
-    'mustache/10_webhook/Test webhook action with mustache integration',
```

Comment:
https://gist.github.com/pgomulka/753da1ea4cb303d313216ced449a81b1
@jakelandis do you have any ideas why is this test failing? Does the stacktrace or logs (see gist) give you any clues?

Reply:
Another variant of the the types removal. In this case Watcher is writing to ES via a webhook action and using types. Types are incidental to this test, so we could have either removed it in 7.x, thereby fixing it here. However, this is a good use case to implement an arbitrary key/value find/replace that leverages the existing `ReplaceTextual` . I put up a PR over at https://github.com/elastic/elasticsearch/pull/75193 for the fix.